src/main/java/hng_java_boilerplate/user/controller/UserController.java

```diff
@@ -12,13 +10,12 @@
 import org.springframework.web.bind.annotation.RestController;
 
 import javax.crypto.BadPaddingException;
-import java.util.Optional;
 
 @RestController
 @RequiredArgsConstructor
 @RequestMapping("/api/v1/users")
 public class UserController {
-    private final UserService userService;
+    private final UserServiceImpl userService;
```

Comment:
Why did you rename the `UserService`?

Reply:
He didn't have interface class so when I changed the UserService class to an interface, I transferred the initial codes in the UserService class into UserServiceImpl class which implements the UserService interface